lib/omniship/newgistics/track/package.rb

```diff
@@ -1,29 +1,14 @@
 module Omniship
   module Newgistics
     module Track
-      class Package
+      class Package < Omniship::Base
+        ARRIVAL_STATUSES = ["Delivered"]
+        DEPARTURE_STATUSES = ["Received", "InTransit", "Departed", "InUSPSNetwork"] + ARRIVAL_STATUSES
```

Comment:
oh. I think thats because newgistics only has 1 status field. All of the other ones look at all activity so they'll inherently have both a departure and arrival.